src/events/ResourceLinks.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Tooltip } from "@patternfly/react-core";
+
+import type AdminEventRepresentation from "@keycloak/keycloak-admin-client/lib/defs/adminEventRepresentation";
+import { useRealm } from "../context/realm-context/RealmContext";
+import { toClient } from "../clients/routes/Client";
+import { toGroups } from "../groups/routes/Groups";
+import { toClientScope } from "../client-scopes/routes/ClientScope";
+import { toUser } from "../user/routes/User";
+import { toRealmRole } from "../realm-roles/routes/RealmRole";
+import { toFlow } from "../authentication/routes/Flow";
+
+type ResourceLinkProps = {
+  event: AdminEventRepresentation;
+};
+
+const MAX_TEXT_LENGTH = 38;
+const Truncate = ({
+  text,
+  children,
+}: {
+  text?: string;
+  children: (text: string) => any;
+}) => {
+  const definedText = text || "";
+  const needsTruncation = definedText.length > MAX_TEXT_LENGTH;
+  const truncatedText = definedText.substring(0, MAX_TEXT_LENGTH);
+  return (
+    <>
+      {needsTruncation && (
+        <Tooltip content={text}>{children(truncatedText + "...")}</Tooltip>
```

Comment:
There is a special character that you can use for the three dots.
```suggestion
        <Tooltip content={text}>{children(truncatedText)}&hellip;</Tooltip>
```

Reply:
This way the `...` is no longer part of the href

Reply:
Ahh, yeah you are right. I updated the suggestion to use the unicode character instead.